mu4/cloud/internal/cloudmanager_p.h

```diff
@@ -31,7 +31,7 @@ class ApiInfo
 {
     static ApiInfo* _instance;
 
-    ApiInfo(QByteArray clientId, QByteArray apiKey);
+    ApiInfo(QByteArray CLIENT_ID, QByteArray API_KEY);
```

Comment:
Why are the names here in capital letters?

Reply:
fixed